Classes/Editor/MultiEditor/MultiEditorViewController.swift

```diff
@@ -68,6 +77,9 @@ class MultiEditorViewController: UIViewController {
     
     private let settings: CameraSettings
 
+    struct Edit {
```

Comment:
This will include more later on, like the setting to mute sound, or other metadata about a frame.